lib/elasticache_redis.py

```diff
@@ -0,0 +1,599 @@
+"""
+    https://aws.amazon.com/blogs/database/monitoring-best-practices-with-amazon-elasticache-for-redis-using-amazon-cloudwatch/
+"""
+
+from typing import List
+from grafanalib.core import (
+    Alert,
+    AlertCondition,
+    Dashboard,
+    LowerThan,
+    Graph,
+    GreaterThan,
+    MILLISECONDS_FORMAT,
+    OP_OR,
+    PERCENT_FORMAT,
+    Repeat,
+    Row,
+    RTYPE_MAX,
+    SHORT_FORMAT,
+    Target,
+    Template,
+    Templating,
+    TimeRange,
+    single_y_axis,
```

Comment:
Can be please have the Current connections as a connected line graph?


![Screen Shot 2021-04-22 at 9 22 56 AM](https://user-images.githubusercontent.com/5619473/115672928-55f69f00-a34c-11eb-9932-f8016d08261e.png)